app/views/pages/_editor_guide_text.html.erb

```diff
@@ -113,369 +115,116 @@
-    <p>You can also use the slug like this:</p>
-    <pre>{% link kazz/boost-your-productivity-using-markdown-1be %}</pre>
-    <p>You can also use the alias post instead of link like this:</p>
-    <pre>{% post <%= app_url("/kazz/boost-your-productivity-using-markdown-1be") %> %}</pre>
-    <p>or this:</p>
-    <pre>{% post kazz/boost-your-productivity-using-markdown-1be %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "user-embed", title: "#{community_name} User Embed" } %>
-    <p>All you need is the <%= community_name %> username:</p>
-    <pre>{% user jess %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "tag-embed", title: "#{community_name} Tag Embed" } %>
-    <p>All you need is the tag name:</p>
-    <pre>{% tag git %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "comment-embed", title: "#{community_name} Comment Embed" } %>
-    <p>All you need is the
-      <code>ID</code> at the end of a comment URL. To get the comment link, click either the timestamp or the menu button in the top right corner on a comment and then click "Permalink". Here's an example:
+      We support native <a href="https://shopify.github.io/liquid" target="_blank" rel="noopener">Liquid tags</a> in our editor, but have created our own custom tags as well. A list of supported custom tags appears below. To create a custom embed, use the complete URL:
+      <code>{% embed Liquid-tag-URL %}</code>.
     </p>
-    <pre>{% comment 2d1a %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "podcast-episode-embed", title: "#{community_name} Podcast Episode Embed" } %>
-    <p>All you need is the full link of the podcast episode:</p>
-    <pre>{% podcast <%= app_url("/basecspodcast/s2e2--queues-irl") %> %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "listing-embed", title: "#{community_name} Listing Embed" } %>
-    <p>All you need is the full link of the listing:</p>
-    <pre>{% listing <%= app_url("/listings/collabs/dev-is-open-source-823") %> %}</pre>
-    <p>You can also use the category and slug like this:</p>
-    <pre>{% listing collabs/dev-is-open-source-823 %}</pre>
-    <p>Note: Expired listings will raise an error. Make sure the listing is published or recently bumped.</p>
 
-    <%= render partial: "editor_guide_h3", locals: { id: "details-embed", title: "Details Embed" } %>
-    <p>You can embed a details HTML element by using details, spoiler, or
-    collapsible. The <em>summary</em> will be what the dropdown title displays.
-    The <em>content</em> will be the text hidden behind the dropdown. This is
-    great for when you want to hide text (i.e. answers to questions) behind a
-    user action/intent (i.e. a click).</p>
+    <%= render partial: "editor_guide_h3", locals: { id: "custom-embeds-list", title: "Supported Custom Liquid Embeds" } %>
+    <p>Note: Old liquid tag embeds will still work.</p>
+    <table class="crayons-table" width="100%">
```

Comment:
I'm confused about the addition of a table.  This doesn't quite look like tabular HTML data.  My read is that we want a two column list.  Is that the intention?

(Also this is almost non-blocker from me but may be something @aitchiss or other folks might have perspective to give)

Reply:
Ah I see your point, re: the use of a table. I'm not sure how to render the two-column list.